mitodl/mitxonline
# add homepage header hero contents

#### Pre-Flight checklist


#### What are the relevant tickets?
fixes #14 

#### What's this PR do?
- Adds the design for the HomePage hero image
- Adds title and subtitle fields in HomePage
- Fixes a crash when there is no Hero Image added in CMS
- Adds design for title and subtitles on HomePage

**Note:**
- Font Used: Roboto, I think we will need a purchased `Avenir Next` font?
- Titles left spacing is based on the container to keep it responsive with the overall website
- Kept title/subtitle fields optional so if they aren't added in CMS there will be no title/subtitle section over Hero image.

#### How should this be manually tested?
- Set up the mix online
- Run `configure_wagtail` command and make sure that Hero title/subtitle is added to Home Page
- Check that the Home Page loads fine and doesn't crash if there is no Hero image set up in CMS
- Check that the design matches

#### Where should the reviewer start?
- Home Page


#### Screenshots (if appropriate)

**Desktop**
<img width="1440" alt="Screenshot 2021-07-29 at 11 39 32 AM" src="https://user-images.githubusercontent.com/34372316/127444950-6aa182c5-e523-499a-86a2-230cb5f79296.png">

**Mobile**
<img width="364" alt="Screenshot 2021-07-29 at 11 40 50 AM" src="https://user-images.githubusercontent.com/34372316/127444961-1eb9a278-660f-4c27-a38b-e99337243a15.png">

**Wagtail UI & Image dimensions helper message**
<img width="1241" alt="Screenshot 2021-07-29 at 11 39 12 AM" src="https://user-images.githubusercontent.com/34372316/127444935-c928d5a9-27a6-4c62-b7f9-34a7afd2b9db.png">

**Wagtail UI Image upload dialog**

<img width="1229" alt="Screenshot 2021-07-29 at 11 41 11 AM" src="https://user-images.githubusercontent.com/34372316/127444965-d6af7fc6-daf5-47a3-9937-864830d0c7b7.png">
